src/detect-pangram.js

```diff
@@ -15,7 +15,7 @@
  * @return {boolean}
  */
 function detectPangram(phrase) {
-  // write code here
+  return new Set(phrase.toLowerCase().replace(/[^a-z]/g, '')).size === 26;
```

Comment:
можно добавить флаг `i` - ignoreCase
```suggestion
  return new Set(phrase.replace(/[^a-z]/gi, '')).size === 26;
```